dev/MRTCore/mrt/NuGet.Config

```diff
@@ -7,5 +7,13 @@
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="TAEF" value="https://pkgs.dev.azure.com/ms/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json" />
+    <add key="winui" value="https://www.nuget.org/packages/Microsoft.WinUI" />
+    <add key="mrtcore" value="https://mscodehub.visualstudio.com/mrtcore/_packaging?_a=feed%26feed=MRTCore" />
+    <add key="cppwinrt" value="https://www.nuget.org/packages/Microsoft.Windows.CsWinRT/" />
```

Comment:
This is private so it would mean that unauthenticated users (e.g. contributors) couldn't use this.